### DISCUSSION TITLE
An issue not a bug or feature request?

### DISCUSSION THREAD

**Discussion by tisonkun** [apache/skywalking#7978] [General]
I'm going to create an enhancement task for skywalking-eyes, but the template is either bug or feature while cannot create a blank issue.

Do we consider this kind of issues "feature request"? I'll create it as a feature request and back to this discussion.

**Comment by tisonkun** [comment]
Here is the issue https://github.com/apache/skywalking/issues/7979

**Comment by wu-sheng** [comment]
From our perspective, feature request includes the `enhancement`. We don't set a very strict rule about this. Basically, about feature requests, we would ask the submitted `whether do you want to submit a pull request to do`.  Generally, SkyWalking PMC don't take care of this case, we rely on the whole community to do by themselves.
PMC focus on CVEs, PR review and their own interests only.

**Reply by tisonkun** [reply]
> feature request includes the enhancement

Fine.
